src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -119,21 +118,21 @@ export class HiGlassViewConfigTabView extends React.PureComponent {
                 'genome_assembly'    : (nextProps.context && nextProps.context.genome_assembly) || this.state.genome_assembly || null
             });
         }
-
-        if (nextProps.href !== this.props.href && object.itemUtil.atId(nextProps.context) === object.itemUtil.atId(this.props.context)){
+        const hiGlassTabIndex = (this.props.hiGlassTabIndex !== 'undefined') ? this.props.hiGlassTabIndex : -1;        
+        const recentTabIsHiglass = (hiGlassTabIndex === 0 ? this.props.href.indexOf('#') < 0 : false) || (this.props.href.indexOf('#higlass') >= 0);
+        if (recentTabIsHiglass && (nextProps.href !== this.props.href) && (object.itemUtil.atId(nextProps.context) === object.itemUtil.atId(this.props.context))) {
             // If component is still same instance, then is likely that we're changing
             // the URI hash as a consequence of changing tabs --or-- reloading current context due to change in session, etc.
             // Export & save viewConfig from HiGlassComponent internal state to our own to preserve contents.
-            var hgc                 = this.getHiGlassComponent(),
-                currentViewConfStr  = hgc && hgc.api.exportAsViewConfString(),
-                currentViewConf     = currentViewConfStr && JSON.parse(currentViewConfStr);
-
-            currentViewConf && _.extend(nextState, {
-                'viewConfig' : currentViewConf
-            });
+            var hgc = this.getHiGlassComponent(),
```

Comment:
_(Not overly important - more for later)_
For most part we can refactor declarations of `var` to `const` (if doesn't get reassigned) or `let` (if does).
(Objects/Arrays whose contents changed but don't get reassigned would fall under `const`).

_(Not overly important, but maybe consider changing if seems simple enough or have time)_
`componentWillReceiveProps` is now deprecated as of React 16 due to async/batched updates and may not always work correctly esp. if many updates having in small amount of time, so if possible we should migrate to `componentDidUpdate(pastProps, pastState)` &/or `static getDerivedStateFromProps(currProps, currState)`